src/collectives.cc

```diff
@@ -79,6 +79,8 @@ const char* ncclProtoToString(int proto) {
   }
 }
 
+RCCL_PARAM(DirectAllGatherThreshold, "DIRECT_ALLGATHER_THRESHOLD", 75497472);
```

Comment:
can you merge develop to your branch? AG detection and env have been moved to rccl_wrap.cc. Not sure why diff is not picking this
https://github.com/ROCm/rccl/blob/0dd2b2f65e745566e6feecc91b6eb1e269610959/src/collectives.cc#L111